pkg/virt-controller/watch/replicaset.go

```diff
@@ -320,11 +320,11 @@ func (c *VMIReplicaSet) scale(rs *virtv1.VirtualMachineInstanceReplicaSet, vmis
 				vmi, err := c.clientset.VirtualMachineInstance(rs.ObjectMeta.Namespace).Create(vmi)
 				if err != nil {
 					c.expectations.CreationObserved(rsKey)
-					c.recorder.Eventf(rs, k8score.EventTypeWarning, FailedCreateVirtualMachineReason, "Error creating virtual machine: %v", err)
+					c.recorder.Eventf(rs, k8score.EventTypeWarning, FailedCreateVirtualMachineReason, "Error creating the instance of virtual machine: %v", err)
```

Comment:
Can you change this to `Error creating the virtual machine instance`, in order to have it consistent with all other places, please?

Reply:
@slintes Fixed